doc/ApiDocGeneration/Generate-Api-Docs.ps1

```diff
@@ -0,0 +1,110 @@
+# Generates API Docs from the Azure-SDK-for-NET repo
+ 
+[CmdletBinding()]
+Param (
+    $ArtifactName,
+    $ServiceDirectory,
+    $ArtifactsDirectoryName,
+    $LibType,
+    $RepoRoot,
+    $BinDirectory
+)
+
+Write-Verbose "Create variables for identifying package location and package safe names"
+$PackageLocation = "$($ServiceDirectory)/$($ArtifactName)"
+Write-Verbose "Package Location $($PackageLocation)"
+
+if ($ServiceDirectory -eq '*') {
+    $PackageLocation = "core/$($ArtifactName)"
```

Comment:
Why can we not set ArtifactDirectoryName in the core\ci.yml and use the same logic for both cogservices and mgmt?

Reply:
We can. But i'll have to set ArtifactDirectoryName as core/artifact name. Then also update the ci.yml in cognitiveservice by including the service name each place I pass the variable.

Reply:
Oh yes they are slightly different cases, we have this same issue in JS repo. For now leave it as you have it but we will want to figure out some logic to avoid this special casing later.